jsonrpc/utils/test/payloadId.test.ts

```diff
@@ -25,10 +25,6 @@ describe("Payload Id", () => {
     chai.expect(payloadId().toString().length).to.equal(16);
   });
 
-  it("returns a bigint with 19 digits", () => {
```

Comment:
Why are you removing this?

Reply:
I moved `bigint` case into a separate test case. The check for length is there: https://github.com/WalletConnect/walletconnect-utils/pull/168/files#diff-2efdab1819dc17dfb490670403c52da588f22d53c99fb710b27848aada10e1acR81-R83